common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -134,15 +111,6 @@ def setUp(self):
         super().setUp()
         self.user_id = ModuleStoreEnum.UserID.test
 
-        self.store = MixedModuleStore(
```

Comment:
Question: It's okay that `self.store` is no longer being set?

Reply:
I think so, because the new parent class (ModuleStoreTestCase) mixes in `ModuleStoreIsolationMixin` [which sets it as a class attribute `cls.store`](https://github.com/edx/edx-platform/blob/b7a34887c234d6598fe53186f90293e66450e886/common/lib/xmodule/xmodule/modulestore/tests/django_utils.py#L311). So any tests that read `self.store` still get the modulestore.